### DISCUSSION TITLE
Level Up - Gamification + others

### DISCUSSION THREAD

**Discussion by cjmellor** [HassanZahirnia/laravel-package-ocean#114] [Package Suggestions]
Hi,

I'd like to put forward a package of mine for consideration. 

- https://github.com/cjmellor/level-up 

Unrelated, but I have a few more:

- https://github.com/cjmellor/rating
- https://github.com/cjmellor/blockade

But I can add these as separate discussions topics if you desire. 

Thanks

**Comment by HassanZahirnia** [comment]
Thanks for the suggestions 😊 I've added the packages!
